rest/model/pod.go

```diff
@@ -0,0 +1,77 @@
+package model
+
+import (
+	"time"
+
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/model/pod"
+	"github.com/evergreen-ci/utility"
+)
+
+// APISecretOpts is the model for secrets in a container.
+type APISecretOpts struct {
+	Name   *string
+	Value  *string
+	Owned  *bool
+	Exists *bool
```

Comment:
It's fine to assume that exists=false and owned=true since we're just using this as a testing route, so we don't need these fields. We'll just set exists=false and owned=true when the pod is being created.